scim/resource_group_member.go

```diff
@@ -3,21 +3,127 @@ package scim
 import (
 	"context"
 	"fmt"
+	"strings"
+	"sync"
 
 	"github.com/databricks/databricks-sdk-go/apierr"
 	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
 )
 
+type GroupMembersInfo struct {
```

Comment:
Do these types need to be exported?
```suggestion
// groupMembersInfo is the set of members that belong to a group. Read and write access
// to the underlying members map must be protected by the lock.
type groupMembersInfo struct {
```